### PR TITLE
Thresholds

### PR DESCRIPTION
**Pull request by MacKarlita**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -36,8 +36,9 @@ var ViewPager = require('react-native-viewpager');
 * **`isLoop`**: `true` to run in infinite scroll mode,
 * **`locked`**: `true` to disable touch scroll,
 * **`onChangePage`**: page change callback,
-* **`renderPageIndicator`**: render custom ViewPager indicator.
-* **`initialPage`**: show initially some other page than first page.
```
Comment (outofculture) [inline README.md]: `initialPage` isn't allowed anymore?

File: ViewPager.js [hunk 2]
```diff
@@ -126,6 +130,13 @@ var ViewPager = React.createClass({
     }
   },
 
+  /* Detect horizontal swipe but with a range of 20 dx points */
+  isHorizontalSwipe(_dx, _dy) {
```
Comment (outofculture) [inline ViewPager.js]: I would say the comment is overkill; the function is already very well named, and now the `20` is hard-coded in both the comment and the logic.

File: README.md (hunk 1 above)
Comment (MacKarlita) [inline README.md]: For this I'm not sure. This change was already on the branch before the changes we did =/ We can track the author of this change and ask the reasons, or readd those options.

Comment (MacKarlita) [inline README.md]: Investigation done. These changes were pulled from any remote branch (Merge remote-tracking branch 'FMCorz/thresholds' into thresholds) but never merged into master.

File: ViewPager.js (hunk 2 above)
Comment (MacKarlita) [inline ViewPager.js]: Comment removed. Hardcoded value moved to a property.

**Pull request merged**